src/zoid/buttons/component.jsx

```diff
@@ -319,10 +320,10 @@ export const getButtonsComponent : () => ButtonsComponent = memoize(() => {
                     logLatencyInstrumentationPhase({ buttonSessionID: props.buttonSessionID, phase: 'buttons-first-render' });
 
                     return (...args) => {
-                        const venmoExperiment = createVenmoExperiment();
+                        const logoCDNExperiment = getLogoCDNExperiment();
 
-                        if (venmoExperiment) {
```

Comment:
I think we do want to keep this `venmoExperiment` for now. We can remove it in a separate PR after discussing it more internally.